src/packer.json

```diff
@@ -29,13 +29,13 @@
       "region_kms_key_ids": {},
       "source_ami_filter": {
         "filters": {
-          "name": "debian-10-amd64-*",
+          "name": "debian-11-amd64-*",
           "root-device-type": "ebs",
           "virtualization-type": "hvm"
         },
         "most_recent": true,
         "owners": [
-          "136693071363"
+          "903794441882"
```

Comment:
This is unexpected.  I didn't expect the owner of the AMI to change.  Did you have to get it from someone else because Bullseye hasn't been released yet by the official Debian AMI source?

Reply:
I got that account ID from [here](https://wiki.debian.org/Cloud/AmazonEC2Image/Bullseye), so I'm pretty sure it's legit.  Not sure if they will change it after Bullseye becomes the stable release.